README.md

```diff
@@ -56,7 +56,7 @@ Define the type of `{{ . }}` in the context of your template by adding a line li
 ## Run Cypress tests for M1 chips
 
 `make down - in sirius repo`
-`docker-compose -f docker/docker-compose.ci.yml up app json-server --build`
+`docker compose -f docker/docker-compose.ci.yml up app json-server --build`
```

Comment:
Is there a reason the hyphen has been removed here but not in line 25 above?

Reply:
Yeah with it in there is an error for m1 chips being `WARNING: Compose V1 is no longer supported and will be removed from Docker Desktop in an upcoming release. See https://docs.docker.com/go/compose-v1-eol/` but without it it does build

Reply:
I think that's going to be true for everyone, as that's a Docker Desktop thing not an M1 chip thing, but you can't test that out yourself as part of this PR.